app/helpers/payment_mailer_helper.rb

```diff
@@ -0,0 +1,27 @@
+module PaymentMailerHelper
+  def breakdown_of_payment_bullets(payment)
+    bullets = []
+
+    !payment.claims.one? && payment.claims.each do |claim|
+      bullets << "#{I18n.t("#{claim.policy.locale_key}.claim_amount_description")}: #{number_to_currency(claim.award_amount)}"
```

Comment:
Do we need `claim_amount_description`s for IRP and FE?
And maybe a spec that checks `Policies.all.map{|p| I18n.t("#{p.locale_key}.claim_amount_description") }`